circle.yml

```diff
@@ -0,0 +1,62 @@
+# Setup in CircleCI account the following ENV variables:
+# BINTRAY_ORGANIZATION
+# BINTRAY_ACCOUNT
+# BINTRAY_API_KEY
+general:
+  artifacts:
+    - ~/packages
+
+machine:
+  environment:
+    DISTROS: "wheezy jessie trusty el7"
```

Comment:
Add also `el6`, we have that repo.

In next few days we'll start shipping RHEL6/CentOS6 packages in https://bintray.com/stackstorm/el6_staging